Add written intro after the infographic (#179)
zkSNACKs/WasabiDoc: docs/why-wasabi/README.md

And why Wasabi can make Bitcoin to an even better tool of self defense by default.

![](/InfographicWhyWasabi.png)

## Gaining privacy with Wasabi Wallet for Bitcoin transactions

> ### The peirceived problem

Reviewer: ```suggestion
### The perceived problem
```

Bitcoin transactions are somewhat *traceable* and this creates the need for a wallet that specializes in privacy.

*_"What kind of privacy? I'm not sure if I need that."_*

> When we make traditional financial transactions (banks, PayPal, etc.), no one but the service provider is able to track our spendings habits or account balances.

Reviewer: ```suggestion
When we make traditional financial transactions (Banks, PayPal, etc.), no one but the service provider is able to track our spending habits or account balances.
```

### Wasabi Wallet

> Wasabi Wallet is an *open source* and *non-custodial* desktop wallet offered by zkSNACKs Ltd.

Reviewer: ```suggestion
Wasabi Wallet is an *open source*, *non-custodial*, *privacy-focused*, desktop Bitcoin wallet offered by zkSNACKs Ltd.
```

> it differentiates itself from many other wallets for its strong *focus on user privacy*.

Reviewer: ```suggestion
It differentiates itself from many other wallets for its strong *focus on user privacy*.
```
